### ISSUE TITLE
Package Into App

### ISSUE THREAD

**Issue by frankobjank** [frankobjank/natac#20]
Package the game in an app so players do not need to download the requirements separately. Also would be nice to have an alternative to running it from the command line.

**Comment by frankobjank** [comment]
Dependent on #16 - currently username and IP address are set via command line. Need to add ability to enter IP address and username after starting app.

**Comment by frankobjank** [comment]
As of commit 893284c IP address and name input are included in the program so client can now be packaged in an executable
